### PR TITLE
Azure managed Prometheus doc update

### PR DESCRIPTION
**Pull request by raggupta-ms**
1. adding info about what persmission the identity needs to access Azure Monitor Workspace
2. updating azure managed prometheus service name to reflect the official service name.
3. removing other clouds (other than Azure public cloud) support since managed prometheus service is only present in public cloud as of now. When it will be preesnt in other clouds, will update the doc.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.10/scalers/prometheus.md [hunk 1]
```diff
@@ -76,13 +70,12 @@ You can use `TriggerAuthentication` CRD to configure the authentication. It is p
 
 > 💡 **NOTE:**It's also possible to set the CA certificate regardless of the selected `authModes` (also without any authentication). This might be useful if you are using an enterprise CA.
 
-**Azure managed service for Prometheus**
+**Azure monitor managed service for Prometheus**
```
Comment (tomkerkhove) [inline content/docs/2.10/scalers/prometheus.md:73]: Nit - It's a product name
```suggestion
**Azure Monitor managed service for Prometheus**
```

**Review by tomkerkhove — COMMENTED** [review]
LGTM, but can you also update v2.11 please?

**Comment by raggupta-ms** [comment]
> LGTM, but can you also update v2.11 please?

not sure if we want that. Treat this doc update as a bug in the documentation for KEDA 2.10. Now I am fixing this doc bug. There are no code changes assosiated with this doc update and the updated documentation is valid for KEDA 2.10.
Does that make sense?

File: content/docs/2.10/scalers/prometheus.md (hunk 1 above)
Comment (raggupta-ms) [reply content/docs/2.10/scalers/prometheus.md]: yes, updated.

**Comment by tomkerkhove** [comment]
Not really as everything from 2.10 is also in 2.11 or above so we have to align it, unfortunately

**Comment by raggupta-ms** [comment]
> Not really as everything from 2.10 is also in 2.11 or above so we have to align it, unfortunately

Ahh.. I think I misread your earlier comment. I though you said make this change on 2.11 and not on 2.10. But you said make this change on **2.11 as well.** Makes sense, updated.

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**